crates/lox-io/src/ndm/ocm.rs

```diff
@@ -0,0 +1,1156 @@
+    #[serde(rename = "COV_UNITS")]
+    pub cov_units: Option<String>,
+    #[serde(rename = "covLine")]
+    pub cov_line_list: Vec<String>,
+}
+
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    lox_derive::KvnDeserialize,
+)]
+#[serde(default)]
+pub struct OcmManeuverParametersType {
+    #[serde(rename = "COMMENT")]
+    pub comment_list: Vec<String>,
+    #[serde(rename = "MAN_ID")]
+    pub man_id: String,
+    #[serde(rename = "MAN_PREV_ID")]
+    pub man_prev_id: Option<String>,
+    #[serde(rename = "MAN_NEXT_ID")]
+    pub man_next_id: Option<String>,
+    #[serde(rename = "MAN_BASIS")]
+    pub man_basis: Option<common::ManBasisType>,
+    #[serde(rename = "MAN_BASIS_ID")]
+    pub man_basis_id: Option<String>,
+    #[serde(rename = "MAN_DEVICE_ID")]
+    pub man_device_id: String,
+    #[serde(rename = "MAN_PREV_EPOCH")]
+    pub man_prev_epoch: Option<common::EpochType>,
+    #[serde(rename = "MAN_NEXT_EPOCH")]
+    pub man_next_epoch: Option<common::EpochType>,
+    #[serde(rename = "MAN_PURPOSE")]
+    pub man_purpose: Option<String>,
+    #[serde(rename = "MAN_PRED_SOURCE")]
+    pub man_pred_source: Option<String>,
+    #[serde(rename = "MAN_REF_FRAME")]
+    pub man_ref_frame: String,
+    #[serde(rename = "MAN_FRAME_EPOCH")]
+    pub man_frame_epoch: Option<common::EpochType>,
+    #[serde(rename = "GRAV_ASSIST_NAME")]
+    pub grav_assist_name: Option<String>,
+    #[serde(rename = "DC_TYPE")]
+    pub dc_type: common::ManDcType,
+    #[serde(rename = "DC_WIN_OPEN")]
+    pub dc_win_open: Option<common::EpochType>,
+    #[serde(rename = "DC_WIN_CLOSE")]
+    pub dc_win_close: Option<common::EpochType>,
+    #[serde(rename = "DC_MIN_CYCLES")]
+    pub dc_min_cycles: Option<u64>,
+    #[serde(rename = "DC_MAX_CYCLES")]
+    pub dc_max_cycles: Option<u64>,
+    #[serde(rename = "DC_EXEC_START")]
+    pub dc_exec_start: Option<common::EpochType>,
+    #[serde(rename = "DC_EXEC_STOP")]
+    pub dc_exec_stop: Option<common::EpochType>,
+    #[serde(rename = "DC_REF_TIME")]
+    pub dc_ref_time: Option<common::EpochType>,
+    #[serde(rename = "DC_TIME_PULSE_DURATION")]
+    pub dc_time_pulse_duration: Option<common::DurationType>,
+    #[serde(rename = "DC_TIME_PULSE_PERIOD")]
+    pub dc_time_pulse_period: Option<common::DurationType>,
+    #[serde(rename = "DC_REF_DIR")]
+    pub dc_ref_dir: Option<common::Vec3Double>,
+    #[serde(rename = "DC_BODY_FRAME")]
+    pub dc_body_frame: Option<String>,
+    #[serde(rename = "DC_BODY_TRIGGER")]
+    pub dc_body_trigger: Option<common::Vec3Double>,
+    #[serde(rename = "DC_PA_START_ANGLE")]
+    pub dc_pa_start_angle: Option<common::AngleType>,
+    #[serde(rename = "DC_PA_STOP_ANGLE")]
+    pub dc_pa_stop_angle: Option<common::AngleType>,
+    #[serde(rename = "MAN_COMPOSITION")]
+    pub man_composition: String,
+    #[serde(rename = "MAN_UNITS")]
+    pub man_units: Option<String>,
```

Comment:
I've noticed we have dedicated unit types elsewhere. Are they not appropriate for this field?

Reply:
The unit types are never part of a bigger type, but they are only part of value/unit types, which is defined as such in the schema. In this case there's no value/unit type defined by the spec. T